src/browser/pages/vscode.ts

```diff
@@ -5,18 +5,54 @@ const options = getOptions()
 // TODO: Add proper types.
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-let nlsConfig: any
+export const nlsConfigElementId = "vscode-remote-nls-configuration"
+
+type NlsConfiguration = {
+  locale: string
+  availableLanguages: { [key: string]: string } | {}
+  _languagePackId?: string
+  _translationsConfigFile?: string
+  _cacheRoot?: string
+  _resolvedLanguagePackCoreLocation?: string
+  _corruptedFile?: string
+  _languagePackSupport?: boolean
+  loadBundle?: any
+}
+
+/**
+ * A helper function to get the NLS Configuration settings.
+ *
+ * This is used by VSCode for localizations (i.e. changing
+ * the display language).
+ **/
+export function getNlsConfiguration(document: Document) {
```

Comment:
How do we feel about this function signature? `document` is available globally in the browser, so it might be silly to pass it in as a prop. Though it does make it a bit easier for testing. Not sure, might be a bikeshed comment.

Reply:
It does feel a bit odd to me since it's an API that is explicitly only for tests, but if the alternative (setting document global in the tests) makes it harder to get coverage then I would say it's worth it.

Reply:
> but if the alternative (setting document global in the tests) makes it harder to get coverage then I would say it's worth it

Yeah, I think it might be so I'll leave as is for now